File: experiments/llm-reasoning-cost/generated_code/go/02_fibonacci_mini.go
```go
package main

import "fmt"

func fib(n int) int {
    if n < 2 {
        return n
    }
    return fib(n-1) + fib(n-2)
}

func main() {
    fmt.Println(fib(0))  // Output: 0
    fmt.Println(fib(1))  // Output: 1
    fmt.Println(fib(5))  // Output: 5
    fmt.Println(fib(10)) // Output: 55
}
```

File: experiments/llm-reasoning-cost/generated_code/c/02_fibonacci_mini.c
```c
#include <stdio.h>

int fib(int n) {
    if (n < 2) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

int main() {
    printf("%d\n", fib(0));  // Output: 0
    printf("%d\n", fib(1));  // Output: 1
    printf("%d\n", fib(5));  // Output: 5
    printf("%d\n", fib(10)); // Output: 55
    return 0;
}
```